bats/tests/k8s/enable-disable-k8s.bats

```diff
@@ -0,0 +1,41 @@
+# Test case 8 & 13
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+verify_k8s_is_running() {
+    wait_for_container_engine
+    run kubectl cluster-info
```

Comment:
What is the point of running this command? You may want to drop the `run` wrapper, so you get a failure when the command fails.

I'll update `scripts/bats-lint.pl` to catch this. I was fooled right now because it sees `status` on the next line. Will require `$status` or `${status` to make it more strict.

Reply:
Dropped the `kubectl cluster-info` call